services/node_manager/src/test/java/com/futurewei/alcor/nodemanager/controller/NodeControllerTest.java

```diff
@@ -275,4 +281,16 @@ public void deleteNodeInfo_invalidId() throws Exception {
             assertTrue(e.getCause().getClass().getSimpleName().contains("ParameterNullOrEmptyException"));
         }
     }
+
+    @Test
+    public void testNodeBulkRegistration() throws Exception {
+        Gson gson=new Gson();
+        String input =
+        "{\"nodeInfos\":[{\"id\":\"c2b79aca-316e-4ce8-a8ac-815e2de1f129\",\"name\":\"compute9\",\"localIp\":\"10.213.43.150\",\"macAddress\":\"90:17:ac:c1:34:5d\",\"veth\":\"eth1\",\"gRPCServerPort\":8080},{\"id\":\"c2b79aca-316e-4ce8-a8ac-815e2de1f120\",\"name\":\"compute10\",\"localIp\":\"10.213.43.151\",\"macAddress\":\"90:17:ac:c1:34:5e\",\"veth\":\"eth1\",\"gRPCServerPort\":8080}]}";
```

Comment:
should the key be "host_infos" or "nodeInfos"?

Reply:
use former one, fixed in 339c752